parachain/pallets/ethereum-beacon-client/Cargo.toml

```diff
@@ -92,4 +91,4 @@ try-runtime = [
 	"pallet-timestamp?/try-runtime",
 	"sp-runtime/try-runtime",
 ]
-beacon-spec-mainnet = []
+fast-runtime = []
```

Comment:
Within the context of this crate, I prefer the name `beacon-spec-minimal`

Reply:
https://github.com/Snowfork/snowbridge/pull/1089#discussion_r1434878747

Reply:
https://github.com/Snowfork/snowbridge/pull/1089/commits/d702ab8a72184fc5cb6fb433b169d0ced142487b